recipes/liburing/all/conanfile.py

```diff
@@ -23,11 +23,13 @@ class Liburing(ConanFile):
         "fPIC": [True, False],
         "shared": [True, False],
         "with_libc": [True, False],
+        "use_libc": [True, False],
     }
     default_options = {
         "fPIC": True,
         "shared": False,
         "with_libc": True,
+        "use_libc": True,
```

Comment:
It seems strange to have two options that (fundamentally) do the same thing but for different versions. I think instead of adding another option, just keep using `with_libc`, even if the name is slightly off.

Reply:
kept `with_libc` and removed `use_libc`